### PR TITLE
Rename setPaymentDisabled to setPaymentStatus

### PR DESCRIPTION
**Pull request by IvanTheGreatDev**
Also changes constant to view app-wide and renamed app-wide from disabled to inactive for the payment status.

#374

### CODE REVIEW

File: apps/finance/artifact.json [hunk 1]
```diff
@@ -984,11 +984,11 @@
 			"notice": "Execute pending payment #`_paymentId`"
 		},
 		{
-			"sig": "setPaymentDisabled(uint256,bool)",
+			"sig": "setPaymentStatus(uint256,bool)",
 			"roles": [
 				"DISABLE_PAYMENTS_ROLE"
 			],
-			"notice": "`_disabled ? 'Disable' : 'Enable'` payment `_paymentId`"
+			"notice": "`_active ? 'Active' : 'Inactive'` payment `_paymentId`"
```
Comment (sohkai) [inline apps/finance/artifact.json]: @izqui I'm thinking we should just remove this file from the repo for now, and add it back when we're getting closer to deployment. We'll have to regen it anyway at that point.

File: apps/finance/contracts/Finance.sol [hunk 2]
```diff
@@ -319,17 +319,17 @@ contract Finance is EtherTokenConstant, IsContract, AragonApp {
...
-    * @notice `_disabled ? 'Disable' : 'Enable'` payment `_paymentId`
+    * @notice `_active ? 'Active' : 'Inactive'` payment `_paymentId`
     * @param _paymentId Identifier for payment
-    * @param _disabled Whether it will be disabled or enabled
+    * @param _active Whether it will be active or inactive
     */
-    function setPaymentDisabled(uint256 _paymentId, bool _disabled)
+    function setPaymentStatus(uint256 _paymentId, bool _active)
         external
         authP(DISABLE_PAYMENTS_ROLE, arr(_paymentId))
```
Comment (sohkai) [inline apps/finance/contracts/Finance.sol]: We should also rename this role, so it's less confusing. It can actually enable or disable existing payments, so I'd rename it to `MANAGE_PAYMENTS_ROLE`.

File: shared/minime/contracts/MiniMeToken.sol [hunk 3]
```diff
@@ -192,7 +192,7 @@ contract MiniMeToken is Controlled {
         require((_to != 0) && (_to != address(this)));
         // If the amount being transfered is more than the balance of the
         //  account the transfer returns false
-        var previousBalanceFrom = balanceOfAt(_from, block.number);
+        uint256 previousBalanceFrom = balanceOfAt(_from, block.number);
```
Comment (sohkai) [inline shared/minime/contracts/MiniMeToken.sol]: Could you revert the changes to the `MiniMeToken` here?

I'm not opposed to fixing the compiler warnings here, but we'd also like it to stay as closely as possible to the original version and we should have a separate discussion if we should just [update to `MMT_0.2`](https://github.com/giveth/minime/blob/master/contracts/MiniMeToken.sol).

File: apps/finance/test/finance.js [hunk 4]
```diff
@@ -667,8 +667,8 @@ contract('Finance App', accounts => {
                 })
             })
 
-            it('fails executing disabled payment', async () => {
-                await finance.setPaymentDisabled(1, true)
+            it('fails executing inactive payment', async () => {
+                await finance.setPaymentStatus(1, false)
```
Comment (sohkai) [inline apps/finance/test/finance.js]: We should also add a test to re-enable a payment and see that it can be executed.

File: shared/minime/contracts/MiniMeToken.sol (hunk 3 above)
Comment (IvanTheGreatDev) [reply shared/minime/contracts/MiniMeToken.sol]: Sure thing!

File: apps/finance/test/finance.js (hunk 4 above)
Comment (IvanTheGreatDev) [reply apps/finance/test/finance.js]: Good idea, I'll add that!

File: apps/finance/artifact.json (hunk 1 above)
Comment (IvanTheGreatDev) [reply apps/finance/artifact.json]: Hmm, I don't know if there's a difference between the two. I kind of feel like enable/disable is better.

Comment (IvanTheGreatDev) [reply apps/finance/artifact.json]: I'd say active/inactive makes more sense if we choose `status` for the name of the function, because saying a payments status is `disable` is weird, but `inactive` sounds totally fine.

Comment (sohkai) [reply apps/finance/artifact.json]: Agreed, thought about it some more and it makes more sense if you can re-enable recurring payments ("active recurring payment" sounds a lot better than "enabled recurring payment").

**Comment by sohkai** [comment]
Just to re-surface this from https://github.com/aragon/aragon-apps/pull/474#discussion_r224405146:

> Although @izqui we should probably think of a way to completely disable a payment at some point (or remove the ability to re-enable payments).

**Comment by IvanTheGreatDev** [comment]
Should be good to go now! 😄

**Comment by IvanTheGreatDev** [comment]
What is this PR waiting on?

**Comment by izqui** [comment]
Just added 51539caa7c1c4ee07584452ae51191b010f018a3 passing whether we are setting the payment as active as a parameter to the role
